LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -206,8 +206,8 @@ private function includePersonDesignatedData(array $srcRecords, array $pdRecords
                     $approver = $dir->lookupEmpUID($dRecords[$v['recordID']]['data']);
                     
                     if (empty($approver[0]['Fname']) && empty($approver[0]['Lname'])) {
-                        $srcRecords[$i]['description'] = $srcRecords[$i]['stepTitle'] . ' (' . $dRecords[$v['recordID']]['name'] . ')';
-                        $srcRecords[$i]['approverName'] = $dRecords[$v['recordID']]['name'];
+                        $srcRecords[$i]['description'] = $srcRecords[$i]['stepTitle'] . ' ( *NEEDS REASSIGNMENT* ' . $dRecords[$v['recordID']]['name'] . ')';

```

Comment:
Thinking 508 compliance.  This _looks_ like it's trying to flag something for a viewer's attention.  Only what if they can't "view"?  Do we want to create a tag system for a "reader" to catch and give audible alert?

Reply:
A screen reader would simply read out the text.

This prepends the text to the step's title when there's no first/last name associated with the approver, so it'll be seen/heard anywhere the stepTitle shows up.